vms/shift/views.py

```diff
@@ -78,7 +78,7 @@ def form_valid(self, form):
                         {'form': form, 'shift_id': shift_id, 'volunteer_id': volunteer_id,}
                     )
             else:
-                messages.add_message(self.request, messages.INFO, 'End time should be greater than start time')

```

Comment:
Why remove the current message? The message 'End time should be greater than start time' itself informs the user that hours such as '23:00 to 1:00 ' cannot be entered. If you do want to additionally display an error message as you have suggested,  you should display it only when the hours are across a day and not for all cases where end time is less than start time